sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosResponse.java

```diff
@@ -26,7 +26,7 @@ protected CosmosResponse(T properties) {
         this.resourceResponseWrapper = null;
     }
 
-    protected CosmosResponse(ResourceResponse resourceResponse, T properties) {
+    protected CosmosResponse(ResourceResponse<?> resourceResponse, T properties) {
```

Comment:
API change for @kushagraThapar and @moderakh to review.

Reply:
This looks good to me.